Tools/CISupport/ews-build/results_db.py

```diff
@@ -33,7 +33,7 @@ class ResultsDatabase(object):
     # TODO: Support more suites (Note, the API we're talking to already does)
     SUITE = 'layout-tests'
     PERCENT_THRESHOLD = 10
-    PERECENT_SUCCESS_RATE_FOR_PRE_EXISTING_FAILURE = 50
+    PERECENT_SUCCESS_RATE_FOR_PRE_EXISTING_FAILURE = 80
```

Comment:
Seems like an improvement; but I'm shocked that we're not using a value closer to 99.

Reply:
99 is a bit dangerous, it would effectively mean "if this test failed in post commit this week at any point, let any failures through EWS"